salt/journal/config/etc-nginx-sites-available-journal.conf

```diff
@@ -57,17 +57,17 @@ server {
     root /srv/journal/web;
 
     {% if pillar.elife.domain == 'elifesciences.org' %}
```

Comment:
Would be useful to cover the old-style article URIs in end2end.

Reply:
This `if` is misleading, `pillar.elife.domain` is always `elifesciences.org` in all environments. So this configuration would be available there (although not redirecting to `https://end2end--cdn-journal.elifesciences.org/`)

Reply:
Ah.

We want to be using the same values as https://github.com/elifesciences/journal-formula/blob/e23915585f969fd9530e7a08e57814b829bf3e71/salt/journal/config/srv-journal-app-config-parameters.yml#L43-L57

Reply:
`pillar.journal.default_host` is the right call, although it is mostly `null` hence this fallback. Should be populated for known environments in `builder-private`.